### PR TITLE
[risk=low][RW-7536] Change background color of text inputs and dropdowns if changed

### PR DESCRIPTION
**Pull request by jmthibault79**
Jan 5 background change

Jan_6_Dropdown_Background

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 1]
```diff
@@ -199,6 +204,8 @@ export const AdminUserProfile = (spinnerProps: WithSpinnerOverlayProps) => {
       const {contactEmail, verifiedInstitutionalAffiliation} = updatedProfile;
       if (changed('contactEmail') || changed('verifiedInstitutionalAffiliation')) {
         await validateAffiliation(contactEmail, verifiedInstitutionalAffiliation?.institutionShortName);
+      } else {
+        clearEmailValidation();
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-profile.tsx:208]: unrelated fix

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 2]
```diff
@@ -274,7 +289,7 @@ export const AdminUserProfile = (spinnerProps: WithSpinnerOverlayProps) => {
         onChangeFreeCreditLimit={(freeTierDollarQuota: number) => updateProfile({freeTierDollarQuota})}
         onChangeInstitution={(institutionShortName: string) => updateInstitution(institutionShortName)}
         onChangeInstitutionalRole={(institutionalRoleEnum: InstitutionalRole) => updateInstitutionalRole(institutionalRoleEnum)}
-        onChangeInstitutionOtherText={(otherText: string) => updateInstitutionalRoleOtherText(otherText.trim())}
+        onChangeInstitutionOtherText={(otherText: string) => updateInstitutionalRoleOtherText(otherText)}
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-profile.tsx:292]: other text can have spaces

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 3]
```diff
@@ -192,13 +205,15 @@ export const AdminUserProfile = (spinnerProps: WithSpinnerOverlayProps) => {
     }
   }
 
-  const changed = (field): boolean => !!getUpdatedProfileValue(oldProfile, updatedProfile, field);
+  const changed = (attributePath): boolean => !!getUpdatedProfileValue(oldProfile, updatedProfile, attributePath);
```
Comment (NehaBroad) [inline ui/src/app/pages/admin/admin-user-profile.tsx:208]: why attributepath and not field?

Comment (jmthibault79) [reply ui/src/app/pages/admin/admin-user-profile.tsx:208]: not sure if path is the right word here, but it now represents an array of fields (I'll add the type!) because it can now be used to compare deeper within the object, like on line 213: 
`changed(['verifiedInstitutionalAffiliation', 'institutionShortName'])`

Comment (jmthibault79) [reply ui/src/app/pages/admin/admin-user-profile.tsx:208]: All the changes from L183 to L216 are part of a bug fix which is not strictly related to the styling change in this PR.  It's in a separate commit so I can easily split it out into a different PR if you prefer

**Review by NehaBroad — APPROVED** [review]

File: ui/src/app/pages/admin/admin-user-profile.tsx (hunk 3 above)
Comment (jmthibault79) [reply ui/src/app/pages/admin/admin-user-profile.tsx]: I did this -> #6116
